test/gen-template/matter/simple-test.zapt

```diff
@@ -19,3 +19,10 @@ ExternalAddon : {{test_external_addon_all_commands_helper}}
 ExternalAddon : {{test_external_addon_all_events_helper}} 
 ExternalAddon : {{test_external_addon_helper}} 
 
+// Extract all baseline types:
```

Comment:
Again, please don't use "baseline" with this stuff.

Reply:
Fixed